tests/common-vars.go

```diff
@@ -11,4 +11,6 @@ var (
 	PodType = "pod"
 	// MultusType defines `pod` resource mapping network type
 	MultusType = "multus"
+	// UnsupportedType defines non-existing, unsupported `unsupported resource mapping network type
```

Comment:
@jakub-dzon should the second "`unsupported" be removed from the description ? also there is a missing comma.

Reply:
The second "unsupported" is the the value used as unsupported type:) I will fix the missing punctuation mark.